packages/common/src/search/wellKnownCatalog.ts

```diff
@@ -0,0 +1,266 @@
+/** This function returns a single IHubCatalog definition JSON object based on the
+ * name and entity type requested
+ * @param i18nScope Translation scope to be interpolated into the catalog
+ * @param name Name of the catalog requested
+ * @param entityType
+ * @param user Owner of the entity, optional but required for certain catalogs
+ * @param collectionNames A list of collection names, optional, if passed in,
+ * only those collections in the catalog will be returned
+ */
+export function getWellKnownCatalog(
+  i18nScope: string,
+  name: WellKnownCatalog,
+  entityType: EntityType,
+  user?: IUser,
+  collectionNames?: WellKnownCollection[]
+): IHubCatalog {
+  switch (entityType) {
+    case "item":
+      return getWellknownItemCatalog(i18nScope, name, user, collectionNames);
+    /* Add other entity handlers here, e.g. getWellknownEventCatalog */
+    default:
+      throw new Error(`Wellknown catalog not implemented for "${entityType}"`);
+  }
+}
+
+/**
+ * @param i18nScope Translation scope to be interpolated into the catalog
+ * @param name Name of the catalog requested
+ * @param user Owner of the entity, optional but required for certain catalogs
+ * @param collectionNames A list of collection names, optional, if passed in,
+ * only those collections in the catalog will be returned
+ * @returns An Item IHubCatalog definition JSON object
+ */
+function getWellknownItemCatalog(
+  i18nScope: string,
+  name: WellKnownCatalog,
+  user?: IUser,
+  collectionNames?: WellKnownCollection[]
+): IHubCatalog {
+  i18nScope = validateI18nScope(i18nScope);
+
+  let catalog;
+  const collections = getWellknownCollections(
+    i18nScope,
+    "item",
+    collectionNames
+  );
+  switch (name) {
+    case "myContent":
+      if (user) {
+        catalog = {
+          schemaVersion: 1,
+          title: `{{${i18nScope}catalog.myContent:translate}}`,
+          scopes: {
+            item: {
+              targetEntity: "item" as EntityType,
+              filters: [{ predicates: [{ owner: user.username }] }],
+            },
+          },
+          collections,
+        };
+      } else {
+        throw new Error(`User needed to get "${name}" catalog`);
+      }
+      break;
+    case "favorites":
+      if (user) {
+        catalog = {
+          schemaVersion: 1,
+          title: `{{${i18nScope}catalog.favorites:translate}}`,
+          scopes: {
+            item: {
+              targetEntity: "item" as EntityType,
+              filters: [{ predicates: [{ group: user.favGroupId }] }],
+            },
+          },
+          collections,
+        };
+      } else {
+        throw new Error(`User needed to get "${name}" catalog`);
+      }
+      break;
+    case "organization":
+      catalog = {
+        schemaVersion: 1,
+        title: `{{${i18nScope}catalog.organization:translate}}`,
+        scopes: {
+          item: {
+            targetEntity: "item" as EntityType,
+            filters: [{ predicates: [{ access: "org" }] }],
```

Comment:
Maybe I'm wrong, but I _think_ this just returns items with access set to the Organization... vs all items the current user can access, in their the org, _regardless of the sharing level_. I'd have expected to see `orgid: user.orgid` or something like that... if this is how we did it before, then continue with it - otherwise check with Jay to confirm expected behavior

Reply:
You are right, we do need to add `orgid: user.orgId` for this filter